src/azul/plugins/metadata/hca/indexer/transform.py

```diff
@@ -293,6 +293,8 @@ def api_schema(self) -> JSON:
 
 accession: Nested = Nested(namespace=null_str, accession=null_str)
 
+atlas: Nested = Nested(atlas=null_str, version=null_str)
```

Comment:
The term "atlas" is already taken in Azul (HCA is an atlas and AnVIL is, too), so we need to disambiguate unless it's evident from the context.

```suggestion
tissue_atlas: Nested = Nested(atlas=null_str, version=null_str)
```